src/model/utils/DateUtils.spec.ts

```diff
@@ -18,6 +18,15 @@ describe("DateUtils", () => {
             // eslint-disable-next-line jest/no-conditional-expect
             return expect(event).toBe(Event.CHRISTMAS);
           }
+          // If Hanukkah falls on the same day as Christmas Eve, then prioritise Christmas Eve
```

Comment:
This block of code can be removed as there aren't any overlapping dates within the test data